### PR TITLE
Interface/email

### PR DESCRIPTION
**Pull request by jthandy**
email analysis: models and subsequent analysis.

### CODE REVIEW

**Comment by jthandy** [comment]
I recognize that not all of this is configured to work with the runner, but I don't think that matters given how much of that we're changing right now. I'd prefer to get this checked into master because it represents basically completed work; I can return to it and update it to fit in with DBT whenever that is complete.

File: analysis/email/analysis.sql [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+  select * from analyst_collective.emails_denormalized
+
+)
+
+select email_number, avg("opened?"::float) as open_rate, avg("clicked?"::float) as ctr,
+  count(*) as num_users
+from emails
+--only look at the first 25 emails someone is sent. should be customized based on business.
+where email_number < 26
```
Comment (drewbanin) [inline analysis/email/analysis.sql:46]: this is kind of troubling

**Comment by drewbanin** [comment]
@jthandy looks good to me if you intend to revisit later

File: analysis/email/analysis.sql (hunk 1 above)
Comment (jthandy) [inline analysis/email/analysis.sql:46]: I don't understand what that means? What is troubling?

**Comment by jthandy** [comment]
merging.

**Pull request merged**